swagger.yaml

```diff
@@ -165,6 +274,13 @@ definitions:
         items:
           $ref: "#/definitions/ResourceObject"
 
+  ServicesResultObject:
+    properties:
+      links:
+        $ref: "#/definitions/Links"
+      data:
+        $ref: "#/definitions/ServicesResourceObject"
+
   ResourceObject:
```

Comment:
You have ResourceObject/ResultObject and ServicesResourceObject/ServicesResultObject. Could you qualify the locations resource and result object? Ex LocationsResultObject.